client/stylesheets/shared/_embed.scss

```diff
@@ -49,7 +49,8 @@
 	}
 
 	#screen-meta-links {
-		position: relative;
+        position: relative;
+        margin-bottom: $header-height + 31px;
```

Comment:
With your other styling fix, this might have to be adjusted (or maybe you kept that in account) I couldn't get my ruler to work haha
But the gap does seem on the bigger side:
<img width="526" alt="Screen Shot 2021-01-22 at 2 34 48 PM" src="https://user-images.githubusercontent.com/2240960/105531105-330e0d80-5cbf-11eb-9390-9255045ecb4e.png">

I think it would be good to add a viewport breakpoint here as well (just for the margin-bottom) given it's not necessary on desktop.

Reply:
thank you @louwie17 👍  I will check on the gap before merging

Reply:
@louwie17 the change is in https://github.com/woocommerce/woocommerce-admin/pull/6118/commits/6fc1d5aca6ad5a4351b5fa9adc0dcd83c4a69ec3. They both have 70x gap between the meta links and the add order button. I will test both of them together locally when https://github.com/woocommerce/woocommerce-admin/pull/6118 gets approval.